src/chi/components/buttons/buttons.scss

```diff
@@ -263,6 +263,10 @@ $sizes: (
         }
       }
 
+      &.-fluid:not(.-float) {
+        width: 100%;
```

Comment:
@davidojedacl Text should be centered by default. `justify-content: center;`

Reply:
@mattnickles why don't we leave left alignment by default and then using utilities classes we can override the left alignment and center the text. And we document it in the button fluid area?

Reply:
@jllr 👍